x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_primary_navigation.tsx

```diff
@@ -51,6 +54,7 @@ export const usePrimaryNavigation = ({
   });
 
   return {
+    canBeCollapsed: !isTourShown,
```

Comment:
If it is possible could you please add some unit test coverage to check that behavior?